### PR TITLE
firefox 38+ supports .urls for RTCIceServer

### PR DESCRIPTION
**Pull request by fippo**
see discussion in issue #18

@alvestrand: not sure if there is any way to really test the constructor. The only thing I can think of is "does it not throw an exception?"

### CODE REVIEW

File: adapter.js [hunk 1]
```diff
@@ -79,55 +100,6 @@ if (navigator.mozGetUserMedia) {
     }, 0);
   };
 
-  // Creates ICE server from the URL for FF.
-  window.createIceServer = function(url, username, password) {
```
Comment (alvestrand) [inline adapter.js:83]: You're withdrawing an utility function. Was this ever used by users? Do we have any way of knowing?

**Comment by alvestrand** [comment]
Sure, you can do some tests. Call "new RTCPeerConnection()" and see that you get back an Object, that has the setLocalDescription function defined. (Because of the nature of the shim, I don't think you can test the actual type of the returned object; it won't be RTCPeerConnection.)
This PR also exposes webrtcMinimumVersion - add a test saying that it has been defined.

I've done so many silly errrors in coding, I always add a really, really basic test.....

File: adapter.js (hunk 1 above)
Comment (fippo) [inline adapter.js:83]: the only user I know of is apprtc here (and it only uses createIceServers) https://github.com/webrtc/apprtc/blob/master/src/web_app/js/util.js#L96-97
-- I had a PR to remove that when updating adapter.

Breaking changes are not good obviously, see issue #19 for that.

**Comment by fippo** [comment]
Good point. Also that allows checking that the detected version is actually supported by adapter.js.

**Comment by alvestrand** [comment]
#18 said that you were adding version numbers here, but I can't see them.
Still, merging. It looks good.

**Pull request merged**